### ISSUE TITLE
Disable Privacy-Preserving Ad Measurement and Telemetry in Firefox

### ISSUE THREAD

**Issue by gnukeith** [pop-os/cosmic-epoch#502]
Modify the default Firefox settings in Pop!_OS to disable privacy-preserving ad measurement and telemetry data collection, enhancing user privacy out-of-the-box.

![image](https://github.com/user-attachments/assets/3de438f0-a1ac-459a-ac16-d0888273dbe4)

**Comment by jokeyrhyme** [comment]
I don't think this GitHub project is for Pop!_OS in general, or for packages within the Pop!_OS Linux distribution

I think it's specifically for the new COSMIC desktop environment, and from your screenshot it seems as if you're running the old GNOME-based COSMIC desktop environment
